.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,4 +15,10 @@ Closes #xxxx (Replace xxxx with the GitHub issue number)
 
 - [ ] Proposed changes are described in the CHANGELOG.md
 
-- [ ] CI Tests succeed and look good!
\ No newline at end of file
+- [ ] CI Tests succeed and look good!
+
+## Requirements after merging
+
+- [ ] Regenerate test resources of task: e.g. all or batch_integration
+
+- [ ] Regenerate dataset resources of task: e.g. all or batch_integration
```

Comment:
I think if we need to regenerate the test resources, we'll also need to regenerate the dataset resources.

```suggestion
- [ ] Need to regenerate `common/` resources

- [ ] Need to regenerate task-specific resources. Specify: <all or ...>
```